setup/aix61/manualBootstrap.md

```diff
@@ -85,60 +122,59 @@ ln -s /opt/freeware/bin/ccache/ccache /opt/freeware/bin/ccache/g++
 ln -s /opt/freeware/bin/ccache/ccache /opt/freeware/bin/ccache/gcc
 cd ../..
 rm -rf tmp
+```
+
+## Install svn
 
-5) install svn, only needed for v8 testing
+*N.B. Only needed for v8 testing*
```

Comment:
Likely yes.  We'd want to remove on one set of machines fist to validate but if we are right we don't need it anymore.